src/borg/item.pyx

```diff
@@ -517,36 +535,37 @@ class ItemDiff:
             return self._item1.chunks == self._item2.chunks
         if self._item1.get_size() != self._item2.get_size():
             return False
-        return ItemDiff._chunk_content_equal(chunk_iterator1, chunk_iterator2)
+        return _chunk_content_equal(chunk_iterator1, chunk_iterator2)
 
-    @staticmethod
-    def _chunk_content_equal(chunks1, chunks2):
-        """
-        Compare chunk content and return True if they are identical.
 
-        The chunks must be given as chunk iterators (like returned by :meth:`.DownloadPipeline.fetch_many`).
-        """
+def _chunk_content_equal(chunks1, chunks2):
```

Comment:
if that's imported from elsewhere, guess we should remove the leading underscore?

Reply:
also, as it is not just comparing one chunk, but multiple, should it be `chunks_contents_equal`?